org/collections/PrimeNumbers.java

```diff
@@ -0,0 +1,42 @@
+package org.collections;
+
+import java.util.HashSet;
+
+public class PrimeNumbers {
+    public static void main(String[] args) {
+        int n = 10;
+        HashSet<Integer> primes = new HashSet<>();
+        for (int i = 2; i < n; i++) {
+            if (isPrime(i)) {
+                primes.add(i);
+            }
+        }
+        HashSet<Integer> allNumbers = new HashSet<>();
+        for (int i = 1; i <= n; i++) {
+            allNumbers.add(i);
+        }
+
+        System.out.println("Prime numbers less than " + n + ":");
```

Comment:
Please use Iterator class instead of fori loop

ex: Iterator<Integer> iterator = primes.iterator();

while (iterator.hasNext()) {
//DO SOMETHING -> find how to get the values from iterator and print them
}